### DISCUSSION TITLE
Setting for default window on launch

### DISCUSSION THREAD

**Discussion by tony** [extrawurst/gitui#1132] [Ideas]
Is this possible to set the commit log/history to launch by default without extra keystrokes?  Is there a configuration variable to set which window is used?

https://user-images.githubusercontent.com/26336/153055541-93c9275f-8cd2-4aae-a238-00092d298fc6.mp4

(If not, let me know if I should change this to a idea / feature request)

**Comment by extrawurst** [comment]
Not supported, no

**Reply by tony** [reply]
@extrawurst Can I remake this into an idea or feature request?

I think it depends on the user,  for me, the default status view is almost totally empty when I load it.  I thought I was seeing a bug

In my case (QA'ing and reviewing) the history is central.  I'm viewing what someone already has committed (or I've already committed in a feature branch) or even in the trunk.

As for un(staged) files - much of that tends to be in short term memory or committed pretty fast (hopefully?)

**Reply by extrawurst** [reply]
For your use case tig and others are probably currently more suitable anyway due to the lack of support for visualizing the branching structure and the different refs: #81

**Reply by tony** [reply]
Wouldn't it be fair to have a config option to set which tab opens by default?
